web/spacewalk-web.spec

```diff
@@ -48,8 +48,9 @@ BuildArch:      noarch
 Requires(pre):  uyuni-base-common
 BuildRequires:  gettext
 BuildRequires:  make
-BuildRequires:  uyuni-base-common
 BuildRequires:  perl(ExtUtils::MakeMaker)
+BuildRequires:  spacewalk-backend
+BuildRequires:  uyuni-base-common
```

Comment:
This is just a reorder, to make it alphabetical, as requested by the guidelines.